discord/emoji.go

```diff
@@ -21,6 +22,14 @@ type Emoji struct {
 	Available     bool           `json:"available,omitempty"`
 }
 
+// Reaction returns a string used for manipulating with reactions. May be empty if the length of Name is 0
+func (e Emoji) Reaction() string {
+	if len(e.Name) == 0 {
```

Comment:
```suggestion
	if e.Name == ""{
```
Whole `len() == 0` is not wrong
I think checking for an empty string makes more sense